.github/workflows/winget.yml

```diff
@@ -0,0 +1,72 @@
+name: Submit release to the WinGet community repository
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-winget:
+    name: Submit to WinGet repository
+
+    # winget-create is only supported on Windows
+    runs-on: windows-latest
+
+    # winget-create will read the following environment variable to access the GitHub token needed for submitting a PR
+    # See https://aka.ms/winget-create-token
+    env:
+      WINGET_CREATE_GITHUB_TOKEN: ${{ secrets.WINGET_CREATE_GITHUB_TOKEN }}
+
+    # Only submit stable releases
+    if: ${{ !github.event.release.prerelease }}
+    steps:
+      - name: Submit package using wingetcreate
+        run: |
+          # Download latest wingetcreate version
+          curl.exe -JLO https://aka.ms/wingetcreate/latest
+
+          # Run once to avoid output of first time telemetry notice in subsequent 'wingetcreate show' command
+          ./wingetcreate settings | Out-Null
+
+          # Get existing WinGet community repository installers info using wingetcreate
+          $json = (./wingetcreate show Microsoft.PowerShell -i --format json | Select -Skip 1 | ConvertFrom-Json)
```

Comment:
The `Select -Skip 1` is a workaround for:

- https://github.com/microsoft/winget-create/issues/629